dev/cnf/oss_dependencies.maven

```diff
@@ -339,7 +339,7 @@ org.cryptacular:cryptacular:1.2.4
 org.eclipse.birt.runtime.3_7_1:org.apache.xml.serializer:2.7.1
 org.eclipse.birt.runtime:org.w3c.css.sac:1.3.1.v200903091627
 org.eclipse.jdt.core.compiler:ecj:4.3.1
```

Comment:
Why do we still have this old version referenced?

Reply:
Jared (or you) has only pointed out `org.eclipse.jdt.core.compiler:ecj:4.4.2`. 

I'm didn't look this other version, but we look to remove it (perhaps with the 3.26 PR)

Reply:
Or I could try now.  Let me update.